src/nupic/algorithms/anomaly_likelihood.py

```diff
@@ -620,23 +620,24 @@ def nullDistribution(verbosity=0):
 
 def normalProbability(x, distributionParams):
   """
-  Given the normal distribution specified in distributionParams, return
-  the probability of getting samples > x
-  This is essentially the Q-function
+  Given the normal distribution specified by the mean and standard deviation in
+  distributionParams, return the probability of getting samples > x.
+  This is the Q-function: the tail probability of the normal distribution.
+
+  :param distributionParams: dict with 'mean' and 'stdev' of the distribution
   """
-  # Distribution is symmetrical around mean
-  if x < distributionParams["mean"] :
-    xp = 2*distributionParams["mean"] - x
-    return 1.0 - normalProbability(xp, distributionParams)
+  if "mean" not in distributionParams or "stdev" not in distributionParams:
+    raise RuntimeError("Insufficient parameters to specify the distribution.")
 
-  # How many standard deviations above the mean are we, scaled by 10X for table
-  xs = 10*(x - distributionParams["mean"]) / distributionParams["stdev"]
+  if x < distributionParams["mean"]:
+    # Gaussian is symmetrical around mean, so flip to get the tail probability
+    xp = 2 * distributionParams["mean"] - x
+    return 1.0 - normalProbability(xp, distributionParams)
 
-  xs = round(xs)
-  if xs > 70:
-    return 0.0
-  else:
-    return Q[xs]
+  # Calculate the Q function with the complementary error function, explained
+  # here: http://www.gaussianwaves.com/2012/07/q-function-and-error-functions
+  z = (x - distributionParams["mean"]) / distributionParams["stdev"]
+  return 0.5 * math.erfc(z/math.sqrt(2))
```

Comment:
Consider replacing `math.sqrt(2)` with the hard-coded value to avoid the function-call penalty.